sdk/webpubsub/Microsoft.Azure.WebPubSub.AspNetCore/src/Extensions/WebPubSubRequestExtensions.cs

```diff
@@ -305,5 +299,12 @@ private static WebPubSubEventType GetEventType(this string ceType)
                 Constants.ContentTypes.JsonContentType => WebPubSubDataType.Json,
                 _ => throw new ArgumentException($"Invalid content type: {mediaType}")
             };
+
+        private static JsonSerializerOptions CreateSystemTextJsonSerializer()
```

Comment:
Nit - consider moving this to a static method on `ConnectionStatesConverter` so it's in the shared source too?